### PR TITLE
String concat example

### PR DESCRIPTION
**Pull request by crisptrutski**
Tried to reproduce a failure, but could not. Added a test demonstrating that it works (now?)

### CODE REVIEW

**Comment by crisptrutski** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/metabase/macaw/80?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#80** <a href="https://app.graphite.dev/github/pr/metabase/macaw/80?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#79** <a href="https://app.graphite.dev/github/pr/metabase/macaw/79?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#78** <a href="https://app.graphite.dev/github/pr/metabase/macaw/78?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @crisptrutski and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: test/resources/acceptance/string_concat.analysis.edn [hunk 1]
```diff
@@ -0,0 +1 @@
+{:tables [{:table "ra"}]}
```
Comment (piranha) [inline test/resources/acceptance/string_concat.analysis.edn:1]: why `ra`?! 🤯

Comment (crisptrutski) [reply test/resources/acceptance/string_concat.analysis.edn:1]: Not sure what happened here, yeah this file is totally wrong.

Comment (crisptrutski) [reply test/resources/acceptance/string_concat.analysis.edn:1]: The problem is that I'm used to IntelliJ autosaving files 😝

File: test/resources/acceptance/string_concat.analysis.edn [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{:tables         [{:table "t"}]
+ :source-columns [{:table "t" :column "x"}
+                  {:table "t" :column "y"}]
+ :columns        [{:table "t" :column "x" :alias "z"}
+                  {:table "t" :column "y" :alias "z"}]}
```
Comment (tsmacdonald) [inline test/resources/acceptance/string_concat.analysis.edn:5]: Does the text fixture need to be loaded in a test file, or does it (now?) auto-load everything it finds?

**Review by tsmacdonald — APPROVED** [review]
:+1: assuming the test is running with magic

**Comment by crisptrutski** [comment]
### Merge activity

* **Jul 10, 3:02 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/metabase/macaw/80) rebased this pull request as part of a merge.

File: test/resources/acceptance/string_concat.analysis.edn (hunk 2 above)
Comment (crisptrutski) [reply test/resources/acceptance/string_concat.analysis.edn:5]: Yeah there's magic - we loop over all the sql files in this directory, and check whether they have corresponding EDN files. The more EDN files you have, and the more you have in them, the more assertions we autogenerate ✨

Comment (crisptrutski) [reply test/resources/acceptance/string_concat.analysis.edn:5]: See https://github.com/metabase/macaw/blob/master/test/macaw/acceptance_tests.clj#L35-L66

Comment (crisptrutski) [reply test/resources/acceptance/string_concat.analysis.edn:5]: I was curious so just checked - we currently have 45 assertions being generated by 17 fixtures.

**Pull request merged**